rewrite-java-test/src/test/java/org/openrewrite/java/DeleteMethodArgumentTest.java

```diff
@@ -65,7 +63,7 @@ void deleteArgumentsConsecutively() {
         rewriteRun(
           spec -> spec.recipes(
             new DeleteMethodArgument("B foo(int, int, int)", 1),
-            new DeleteMethodArgument("B foo(int, int, int)", 1)
+            new DeleteMethodArgument("B foo(int, int)", 1)
```

Comment:
Since we are updating the methodType, now it is required to match the new resulting "signature"

Reply:
Here is one case where that was required: https://github.com/openrewrite/rewrite-testing-frameworks/commit/47ccd370247f1171fa9df005da8a9a3342d19f3f